### ISSUE TITLE
UTF-8 compatibility

### ISSUE THREAD

**Issue by mabi-travix** [dromara/ChatGPT#107]
# Please give results in UTF-8

Prompt:
can you write german letters ä, ö, ü, Ä, Ö, Ü and ß ?

Result:
As an AI language model, I can certainly write German letters ￤, ￶, ￼, ￄ, ￖ, ￜ and ￟, here they are:
￤ ￄ ￶ ￖ ￼ ￜ ￟

![image](https://user-images.githubusercontent.com/100709655/224355895-0f6cf9a0-1f22-4497-93b0-704f204fe25d.png)

**Comment by obiscr** [comment]
According to my test results, this is due to a problem in JSON conversion, not caused by UTF-8.

ChatGPT Response： (I have formatted it)

```
Yes, I can write German letters with markdown format. Here they are:
- \u00e4: `&auml;` or `&#228;
- \u00f6: `&ouml;` or `&#246;
- ...
```

GPT 3.5 Turbo Response：
```
Yes, sure!
- ä: &auml;
- ö: &ouml;
- ...
```

```
char (ä) == unicode (&#228;) == ascii (\u00e4)
```

**The reason for the garbled code is that the response from GPT 3.5 Turbo is not parsed correctly.**

The next version will be replaced with a new parser. This issue will be fixed.

**Comment by obiscr** [comment]
This issue has been fixed, please update to version 2.1.6.
